apps/backoffice-v2/src/common/components/molecules/ProcessTracker/hooks/useProcessTracker/process-tracker-adapters/third-party.proces-tracker.tsx

```diff
@@ -20,7 +20,8 @@ export class ThirdPartyProcessTracker implements IProcessTracker {
       this.plugins
         ?.filter(({ name }) => pluginsWhiteList.includes(name as (typeof pluginsWhiteList)[number]))
         ?.map(({ displayName, name }) => {
-          const pluginStatus = this.getPluginByName(name)?.status ?? ProcessStatus.DEFAULT;
+          const plugin = this.getPluginByName(name);
```

Comment:
This change made it easier for me to debug the plugin separately from its status.